elk/extensions/recursive_ccs/parser.py

```diff
@@ -0,0 +1,12 @@
+from elk.training.parser import get_training_parser
+from argparse import ArgumentParser
+
+
+def get_rccs_training_parser() -> ArgumentParser:
+    parser = get_training_parser()
+    parser.add_argument(
+        "--num-recursive-iterations",
```

Comment:
Could be shortened to `--num-iterations` or `--num-iter` or `--num-directions`

Reply:
Done!